test/unit/component/parser/json/BpmnJsonParser.sequenceFlow.default.test.ts

```diff
@@ -0,0 +1,691 @@
+                  "sequenceFlow": {
+                      "id": "sequenceFlow_id_0",
+                      "sourceRef": "gateway_id_0",
+                      "targetRef": "targetRef_RLk"
+                  }
+              },
+              "BPMNDiagram": {
+                  "id": "BpmnDiagram_1",
+                  "BPMNPlane": {
+                      "id": "BpmnPlane_1",
+                      "BPMNShape": {
+                          "id":"shape_gateway_id_0",
+                          "bpmnElement":"gateway_id_0",
+                          "Bounds": { "x": 362, "y": 232, "width": 36, "height": 45 }
+                      },
+                      "BPMNEdge": {
+                          "id": "edge_sequenceFlow_id_0",
+                          "bpmnElement": "sequenceFlow_id_0"
+                      }
+                  }
+              }
+          }
+      }`;
+
+    const model = parseJsonAndExpectOnlyEdgesAndFlowNodes(json, 1, 1);
+
+    verifyEdge(model.edges[0], {
+      edgeId: 'edge_sequenceFlow_id_0',
+      bpmnElementId: 'sequenceFlow_id_0',
+      bpmnElementName: undefined,
+      bpmnElementSourceRefId: 'gateway_id_0',
+      bpmnElementTargetRefId: 'targetRef_RLk',
+      bpmnElementKind: SequenceFlowKind.DEFAULT,
+    });
+  });
+
+  it('json containing one process with a sequence flow defined as default in a inclusive gateway', () => {
+    const json = `{
+          "definitions": {
+              "process": {
+                  "id": "Process_1",
+                  "inclusiveGateway": { "id": "gateway_id_0", "default": "sequenceFlow_id_0"},
+                  "sequenceFlow": {
+                      "id": "sequenceFlow_id_0",
+                      "sourceRef": "gateway_id_0",
+                      "targetRef": "targetRef_RLk"
+                  }
+              },
+              "BPMNDiagram": {
+                  "id": "BpmnDiagram_1",
+                  "BPMNPlane": {
+                      "id": "BpmnPlane_1",
+                      "BPMNShape": {
+                          "id":"shape_gateway_id_0",
+                          "bpmnElement":"gateway_id_0",
+                          "Bounds": { "x": 362, "y": 232, "width": 36, "height": 45 }
+                      },
+                      "BPMNEdge": {
+                          "id": "edge_sequenceFlow_id_0",
+                          "bpmnElement": "sequenceFlow_id_0"
+                      }
+                  }
+              }
+          }
+      }`;
+
+    const model = parseJsonAndExpectOnlyEdgesAndFlowNodes(json, 1, 1);
+
+    verifyEdge(model.edges[0], {
+      edgeId: 'edge_sequenceFlow_id_0',
+      bpmnElementId: 'sequenceFlow_id_0',
+      bpmnElementName: undefined,
+      bpmnElementSourceRefId: 'gateway_id_0',
+      bpmnElementTargetRefId: 'targetRef_RLk',
+      bpmnElementKind: SequenceFlowKind.DEFAULT,
+    });
+  });
+
+  // TODO: To uncomment when we support complex gateway
```

Comment:
:warning: to be sure this kind of tests is activate when the related BPMN elements is supported, we have to find a way to better hightlight this for the contributor. Probalby in the the `ShapeBpmnElementKind` where we had the enum value for the newly supported element

Reply:
Done